fly.io/fly.toml

```diff
@@ -0,0 +1,40 @@
+app = "<YOUR APP NAME>"
+
+kill_signal = "SIGINT"
+kill_timeout = 5
+
+[build]
+  image = "jointakahe/takahe-dev:edge"
```

Comment:
Since Takahē is still moving pretty quick, might be reasonable to pin this example to the version it was tested against. Otherwise the user will hit weird issues until 1.0